spec/models/grda_warehouse/system_cohorts/system_cohort_spec.rb

```diff
@@ -0,0 +1,305 @@
+require 'rails_helper'
+
+RSpec.describe GrdaWarehouse::SystemCohorts::CurrentlyHomeless, type: :model do
```

Comment:
Strange that this is here, and the merge is to [ea/system-cohort-changes](https://github.com/greenriver/hmis-warehouse/tree/ea/system-cohort-changes)

Reply:
Yes, I believe I accidentally branched off the wrong branch.  I'll see if I can rebuild the PR, something isn't right.